Resources/Prototypes/_Reserve/Entities/Spawners/ERTSecurity.yml

```
@@ -0,0 +1,41 @@
+- type: randomHumanoidSettings
+  id: ERTSecurityReserve
+  speciesBlacklist: [ Arachnid, Diona, Dwarf, Reptilian, SlimePerson, Vulpkanin, Harpy, Rodentia, Tajaran]
```

губы добавили ещё одно поле в этот тип прототипа
```suggestion
  speciesWhitelist: [Human]
```